src/main/java/Duke.java

```diff
@@ -1,10 +1,287 @@
+            throw new ArrayIndexOutOfBoundsException();
+        }
+
+        //Exception 3: no 'by'
+        if(!commandParamsList[1].startsWith("by ")){
+            throw new DeadlineParamsFormatException();
+        }
+        String todoString = commandParamsList[0];
+        String deadlineString = commandParamsList[1].substring(3);
+
+        Deadline newDeadlineObject = new Deadline(todoString, deadlineString);
+        tasksList.add(newDeadlineObject);
+
+        String feedback =  "Got it. I've added this task:\n"
+                + newDeadlineObject.toString() + "\n"
+                + "Now you have " + tasksList.size() +" tasks in the list";
+        return feedback;
+    }
+
+    public static String executeEventCommand(String commandParams) throws EventParamsFormatException {
+        //input: String /from [startTime] /to [endTime]        [haven't done]: process case like /to /from
+        //Exception 1: No '/'
+        if(commandParams.indexOf('/') == -1){
+            throw new NullPointerException();
+        }
+
+        //Exception 2: Less than 2 '/' or Multiple '/'
+        String[] commandParamsList = commandParams.split("/");
+        if(commandParamsList.length != 3){
+            throw new ArrayIndexOutOfBoundsException();
+        }
+
+        //Exception 3: no 'from' or 'to'
+        String eventString = commandParamsList[0].trim();
+        if(!(commandParamsList[1].startsWith("from ")&&commandParamsList[2].startsWith("to "))){
+            throw new EventParamsFormatException();
+        }
+        String fromString = commandParamsList[1].substring(5).trim();   //magic number
+        String toString = commandParamsList[2].substring(3).trim();
+
+        Event newEventObject = new Event(eventString, fromString, toString);
+        tasksList.add(newEventObject);
+
+        String feedback = "Got it. I've added this task:\n"
+                + newEventObject.toString() + "\n"
+                + "Now you have " + tasksList.size() +" tasks in the list";
+        return feedback;
+    }
+
+    public static String executeCommand(String userCommand){
+        String[] commandTypeAndParams = getCommandTypeAndParams(userCommand);
+        String commandType = commandTypeAndParams[0];
+        String commandParams = commandTypeAndParams[1];
+
+        String feedback = "";
+
+        /*
+        /[have not done]: create methods to deal with input commandParams,
+        instead of dealing them in execute methods.
+        Then execute methods' length can be shorter
+         */
+
+        switch (commandType){
+            case("list"):{
+                if(commandParams != null){
+                    return "ListParamsError: List command do not have Params!";
+                }
+                feedback = executeListCommand(commandParams);
+                break;
+            } case("add"):{
+                feedback = executeAddTaskCommand(commandParams);
+                break;
+            } case("mark"):{
```

Comment:
```suggestion
            case("add"):
                feedback = executeAddTaskCommand(commandParams);
                break;
            case("mark"):{
```
The curly braces are not required in `case` statements, the `break` statements are sufficient